packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.types.ts

```diff
@@ -52,6 +57,11 @@ export interface RadioGroupProps extends Pick<FocusZoneProps, 'isCircularNavigat
    */
   value?: string;
 
+  /**
+   * Sets the RadioGroup to disabled if true
+   */
+  disabled?: boolean;
```

Comment:
disabled should be included as part of IViewProps

Reply:
Is it supposed to be a part of the adapters? I don't see it in IViewProps.